src/cli.ts

```diff
@@ -237,7 +237,7 @@ program
 program.parse();
 
 const options = program.opts();
-const dstPath = './';
+const dstPath = '.';
```

Comment:
This is needed to have `./trace-2024-06-25T08_42_59.123Z.cpuprofile.txt-converted.json` instead of `.//trace-2024-06-25T08_42_59.123Z.cpuprofile.txt-converted.json`